app/controllers/v0/health_care_applications_controller.rb

```diff
@@ -5,6 +5,8 @@ class HealthCareApplicationsController < ApplicationController
     FORM_ID = '1010ez'
 
     skip_before_action(:authenticate)
+    before_action(:append_to_skip_sentry_exception_types)
+    after_action(:remove_from_skip_sentry_exception_types)
```

Comment:
I'm not sure this is a great pattern, it would likely be overlooked and cause headaches for devs. Let's rethink our approach here.